### DISCUSSION TITLE
Add gist to GitHub app

### DISCUSSION THREAD

**Discussion by wilkice** [community/community#5332] [Mobile] [answered]
Wish to see gist on GitHub app,just like code repository

**Comment by alircxa** [comment]
> Wish to see gist on GitHub app,just like code repository

Totally agree!

**Reply by genpro232-arch** [reply]
root

**Reply by genpro232-arch** [reply]
magix

**Comment by elisealix22** [comment, marked as answer]
Hi @wilkice , thank you so much for the request! I've added your note to our internal issue where we are tracking adding gists. If you have any other feedback or feature requests, please keep them coming!

**Reply by 75033us** [reply]
it has been an year, is there any update on adding gist support to the github ios app? thanks.

**Reply by lolwilliams** [reply]
Hi

**Reply by lolwilliams** [reply]
Hi

**Comment by AlexHedley** [comment]
#5614
#5921
#6466

**Comment by kanition** [comment]
Still nothing in 2025

**Comment by gitmpr** [comment]
Is this not a high priority todo item for github? I should create repos instead of gists maybe then

**Reply by TPS** [reply]
Not high priority, b/c their corporate/paying clients aren't clamoring for it 🤷🏾‍♂️

**Comment by sweetcandye** [comment]
https://vnrom.net/bypass/  Ignore it 2.0

**Comment by tomholford** [comment]
In the age of coding agents, gists are super useful. I tell Claude "use gh cli to publish this analysis to a gist" so I can share it with colleagues. Would be great to access the gists on the go.

**Reply by lolwilliams** [reply]
Hi how are you doing

**Comment by montinode** [comment]
Disable Electrode Array, Disable Autosense, Disable Nucleas 8, Disable Nexa, Disable Sonnet *, Disable Naida CI Marvel, Disable Synchrony 2, --SuperDisable Transmitter

**Comment by lolwilliams** [comment]
Hello

**Comment by lolwilliams** [comment]
Hi
